src/openrct2/world/Location.hpp

```diff
@@ -21,6 +21,7 @@ constexpr int32_t kCoordsZStep = 8;
 constexpr int32_t kCoordsZPerTinyZ = 16;
 
 constexpr auto kNumOrthogonalDirections = 4;
+constexpr auto kTrackDirectionDiagonalMask = (1 << 2);
```

Comment:
We tend to write mask in binary.

```suggestion
constexpr auto kTrackDirectionDiagonalMask = 0b0100;
```